wlauto/config_example.py

```diff
@@ -286,4 +286,8 @@
 # Scratch resource getter.
 #scratch_mount_point = '/arm/scratch'
 
+# URL of server from where to fetch external resources for workloads on-demand.
+# Used by WA's get-assets sub-command.
+remote_assets_url = 'https://github.com/jimboatarm/wa-assets/raw/master/dependencies'
```

Comment:
This shouldn't be left empty, as the command is not usable that way.
Please create the appropriate repo under ARM-software account.

On 04 Aug 2016, 04:01:10 -0700, muendelezaji wrote:
> > @@ -286,4 +286,8 @@
> >  # Scratch resource getter.
> >  #scratch_mount_point = '/arm/scratch'
> >
> > +# URL of server from where to fetch external resources for workloads on-demand.
> > +# Used by WA's get-assets sub-command.
> > +remote_assets_url = 'https://github.com/jimboatarm/wa-assets/raw/master/dependencies'
>
> Ok. Since it is now configurable, perhaps we can remove it or keep it empty as a guide in the `config.py`?
>
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/ARM-software/workload-automation/pull/219/files/80193284aecba6b90b881e165f6b51ffb448ea00#r73501766
IMPORTANT NOTICE: The contents of this email and any attachments are confidential and may also be privileged. If you are not the intended recipient, please notify the sender immediately and do not disclose the contents to any other person, use it for any purpose, or store or copy the information in any medium. Thank you.